### PR TITLE
iputils: version bumped to 20150815.

### PR DESCRIPTION
**Pull request by Florin65**
ping6 is now included in ping.
man pages can be optionally regenerated at build.

### CODE REVIEW

**Comment by lunar-linux-b01** [comment]
SUCCESS!

File: net/iputils/DEPENDS [hunk 1]
```diff
@@ -1,3 +1,8 @@
 depends libcap
 depends openssl
 depends sysfsutils
+depends git
+
+if [ "`get_module_config DOCS`" = "y" ]; then
```
Comment (Ratler) [inline net/iputils/DEPENDS]: No logic in the DEPENDS files. Make it an optional_depends and add "if in_depends $MODULE docbook-utils; then ... fi" in BUILD instead.

File: net/iputils/BUILD [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
 
-  ln -sf ping.8.gz /usr/share/man/man8/ping6.8.gz &&
-  ln -sf tracepath.8.gz /usr/share/man/man8/tracepath6.8.gz &&
+gather_docs doc/*.html &&
 
-  gather_docs doc/*.html
-
-) > $C_FIFO 2>&1
+install -dm755 /etc/xinetd.d/ &&
+install -m644 $SCRIPT_DIRECTORY/tftp.xinetd /etc/xinetd.d/tftp
```
Comment (Ratler) [inline net/iputils/BUILD]: @Florin65 optional_depends for xinetd, then put the tftp file in $SCRIPT_DIRECTORY/xinetd.d and the xinetd plugin will install the file.

**Comment by Florin65** [comment]
OK, thanks.

**Pull request closed without merging**